src/app/shared/components/template/components/button/button.component.html

```diff
@@ -47,6 +49,7 @@
         [parent]="parent"
         [attr.data-rowname]="_row.name"
       ></plh-template-component>
+      <ion-ripple-effect></ion-ripple-effect>
```

Comment:
This can be removed – the button itself already has a ripple effect applied by line 39